src/website/app/demos/ButtonGroup/examples/sizes.js

```diff
@@ -0,0 +1,47 @@
+/* @flow */
+import Button from '../../../../../library/Button';
+import ButtonGroup from '../../../../../library/ButtonGroup';
+import DemoLayout from '../../shared/DemoLayout';
+
+export default {
+  id: 'sizes',
+  title: 'Sizes',
+  description: `To provide hierarchy to actions on your page, change ButtonGroup
+  impact with the \`size\` attribute.
+
+  For a ButtonGroup whose width is defined by its container, use \`fullWidth\`.
+`,
+  scope: { Button, ButtonGroup, DemoLayout },
+  source: `() => {
+    const sizes = ['small', 'medium', 'large', 'jumbo'];
+
+    return (
+      <DemoLayout>
+        {
+          sizes.map((size) => (
+            <ButtonGroup
+              size={size}
+              aria-label="Galaxies"
+              defaultChecked={0}
+              key={size}
+              mode="radio">
+              <Button>Andromeda</Button>
+              <Button>Triangulum</Button>
+              <Button>Centaurus</Button>
+            </ButtonGroup>
+          ))
+        }
+        <ButtonGroup
+          fullWidth
```

Comment:
this feels strange to be an example here and a "bad practice" below. what's a good use of this feature? can we put that on display?

Reply:
Not sure if there's a good way to display the better practice, but yes, it is a bit unclear & hard to define. I'll remove it.